README.md

```diff
@@ -7,6 +7,10 @@ Libocpp can be used for the communication of one charging station and multiple E
 
 Libocpp provides a complete implementation of OCPP 1.6. The implementation of OCPP 2.0.1 is currently under development.
 
+## Get Involved
+
+See the [COMMUNITY.md](COMMUNITY.md) and [COMMUNITY.md](COMMUNITY.md) of the EVerest project to get involved.
```

Comment:
These are two links to the same file.

The file is located in a different repo, link needs to be adapted: https://github.com/EVerest/EVerest/blob/main/COMMUNITY.md